File: python/Euklidesgame.py
def euklidesGame(A, B): 
    count = 0

    if A == B: 
        count = A + B
        return count 

    if A > B: 
        difference = lambda A, B: A - B
        count = A - difference(A, B)
        return count

    if B > A: 
        difference = lambda A, B: B - A
        count = B - difference(A, B)
        return count

print(euklidesGame(2, 4))

File: c++/euklidesGame.cpp
#include "pch.h"
#include <iostream>

using namespace std; 


int euklidesGame(int A, int B); 
int main()
{
	cout << euklidesGame(2, 4); 
}
int euklidesGame(int A, int B) {
	int count = 0;

	if (A == B)
	{
		count = A + B;
	}

	if (A > B)
	{
		int difference = A - B;
		count = A - difference;
	}
	if (B > A)
	{
		int difference = B - A;
		count = B - difference;
	}
	return count;
}
